platforms/core-configuration/configuration-cache/src/main/kotlin/org/gradle/internal/cc/impl/fingerprint/ConfigurationCacheFingerprintChecker.kt

```diff
@@ -291,6 +294,14 @@ class ConfigurationCacheFingerprintChecker(private val host: Host) {
                     text("the set of system properties prefixed by ").reference(prefix).text(" has changed: ").text(detailedMessageForChanges(snapshotWithoutIgnored, currentWithoutIgnored))
                 }
             }
+
```

Comment:
❌  This blows up the 100 line limit. I would add: 

`@Suppress("CyclomaticComplexMethod", "LongMethod")`

 instead of somehow avoiding going over the limit.

Reply:
Fixed in a8b51db10ca9d74c02f5a2e6631b6476f3fec0e0